.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "lib/llvm/src"]
+	path = lib/llvm/src
+	url = https://github.com/winksaville/llvm.git
```

Comment:
Can this be pointed toward https://github.com/llvm-mirror/llvm?

Reply:
Actually it should be pointing at https://github.com/ponylang/llvm.git, I'll fix it.